pkg/cmd/resource/operation.go

```diff
@@ -29,6 +29,11 @@ type OperationCmd struct {
 	HTTPVerb  string
 	Path      string
 	URLParams []string
+
+	stringFlags map[string]*string
+	intFlags    map[string]*int
```

Comment:
I think it would be better to just store every scalar parameter as a string, and let the API return an error message when an invalid type is used. The error message returned by cobra/pflag isn't super explicit or helpful:

```
$ go run cmd/stripe/main.go charges create --source=tok_visa --currency=usd -s --amount=abc
invalid argument "abc" for "--amount" flag: strconv.ParseInt: parsing "abc": invalid syntax
exit status 1
```